h2o-algos/src/test/java/hex/glm/GLMTest.java

```diff
@@ -760,6 +763,7 @@ public void testBounds() {
       params._alpha = new double[]{1};
       params._lambda = new double[]{0.001607};
       params._obj_reg = 1.0/380;
+      params._fold_assignment = Model.Parameters.FoldAssignmentScheme.AUTO;
```

Comment:
why are you adding this auto everywhere?

Reply:
because fold assignment is only applicable if a value for nfolds is specified and a fold_column is not specified. Otherwise it is evaluated as null. if you then reuse parameters like this (with fold_assignment = null)for next model building, it creates a problem, because you want to have it set to AUTO (not null) in input so that it can be evaluated in next model computation.

Reply:
enabled null as fold assignment input value